easybuild/easyblocks/f/fsl.py

```diff
@@ -67,46 +67,51 @@ def configure_step(self):
 
         best_cfg = None
 
-        # Patch files for ver. < 5.0.10 patch multiple config directories
-        if LooseVersion(self.version) >= LooseVersion('5.0.10'):
-            # Check if a specific machine type directory is patched
-            systype_regex = re.compile("^diff.*config\/(.*(apple|gnu|i686|linux|spark)(?:(?!\/).)*)", re.M)
-
-            patched_cfgs = []
-
-            for patch in self.patches:
-                patchfile = read_file(patch['path'])
-                res = systype_regex.findall(patchfile)
-                patched_cfgs.extend([i[0] for i in res])
-
-            # Check that at least one config has been found
-            if patched_cfgs:
-                # Check that a single config has been patched
-                if len(nub(patched_cfgs)) == 1:
-                    best_cfg = patched_cfgs[0]
-                    self.log.debug("Found patched config dir: %s", best_cfg)
+        # Predefined makefiles for various configs have disappeared in v6.0.2.
+        # The next part of the EasyBlock has thereby become obsolete.
+        # See https://github.com/easybuilders/easybuild-easyblocks/issues/1859
+        if LooseVersion(self.version) < LooseVersion('6.0.2'):
```

Comment:
How about collapsing both version checks in a single `if` statement (which avoids the need to indent one more level):

```python
loosever = LooseVersion(self.version)
if loosever >= LooseVersion('5.0.10') and loosever < LooseVersion('6.0.2'):

Reply:
The "try" on line 100 in the "new" code is inside the < 6.0.2 but outside >= 5.0.10, so you need them separated.

Reply:
What @akesandgren said :) So... what do I do: click 'resolve conversation', or is that up to you as reviewers? =)

Reply:
OK, I totally overlooked that. Will give this another closer look, hopefully tomorrow.